packages/network-of-terms-graphql/src/schema.ts

```diff
@@ -9,6 +9,7 @@ export const schema = `
     description: String!
     creators: [Creator]!
     features: [Feature]!
+    inLanguage: [String]!
```

Comment:
Made this a list so we can add multiple language support in the future without breaking backwards compatibility in the GraphQL API.